content/blog/2021/09/2021-09-20-election-period-opened.adoc

```diff
@@ -0,0 +1,95 @@
+As mentioned on link:/participate/[jenkins.io.io/participate], they are many different ways to contribute to Jenkins and for many of them, it’s very difficult to measure.
+Therefore we’ll trust participants and will not require that they provide evidence of contribution as part of their voter registration. We reserve the right to ban the specific account from the election process if we identify abuse.
+The election works in three stages:
+
+. Identify voters and nominees
+. Voting period
+. Announce results
+
+## Voters 
+
+To invite participants to vote, we need a list of email’s addresses that we would share with the link:https://civs.cs.cornell.edu[Condorcet Internet Voting Service].
+Therefore we ask every community members who match the requirement to join the group link:https://community.jenkins.io/g/election-voter[election-voter] on link:https://community.jenkins.io[community.jenkins.io] .
+The group will be open for joining during the registration period after we’ll close registration during the voting period.
+We’ll use emails from the “election-voter” group members.
+
+## Nominees
+
+During the same period, we invite every community member to nominate candidates by sending a message to the group link:https://community.jenkins.io/g/election-committee[election-committee] mentioning the position and the motivation.
+On the 31 of October, the nomination period is over. We’ll notify every nominees and get confirmation that they are interested in running as a candidate.
+The list of candidates will be announced on the 7th of November.
+
+**Everybody can nominate candidates.**
+
+This year we are looking for nominees for the following positions:
+
+- link:https://www.jenkins.io/project/governance/#governance-board[Board members]
+- link:https://www.jenkins.io/project/team-leads/#documentation[Documentation Officer]
+- link:https://www.jenkins.io/project/team-leads/#events[Events Officer]
+- link:https://www.jenkins.io/project/team-leads/#infrastructure[Infrastructure Officer]
+- link:https://www.jenkins.io/project/team-leads/#release[Release Officer]
+- link:https://www.jenkins.io/project/team-leads/#security[Security Officer]
+
+More information about the different roles can be found on link:https://www.jenkins.io/project/team-leads/[jenkins.io/project/team-leads].
+
+## Election
+
+On the 7th of November, once voters and candidates are identified, we’ll invite everybody by email to vote using link:https://civs.cs.cornell.edu[civs.cs.cornell.edu].
+At this stage of the election, nobody will be allow to register.
+Voting deadline is the 30th of November.
+
+## Result
+
+As soon as we have the election results, we’ll publish them.
+Elected members will officially be the new officer/board member on the 3rd of December 2021.
+
+## Key Dates
+
+* Sep 20: Nomination and voter registration begin
+* Oct 31: Nomination deadline
+* Nov 07: Candidates announced, Registration deadline, voting start
```

Comment:
I would give more time here. 1 week might not be enough for confirmations and statements if there are many last minute candidate

Reply:
Good insight.
I am considering starting reaching out to nominees before the deadline to reduce the last-minute work.